### DISCUSSION TITLE
Is it possible to auto assign cert to site?

### DISCUSSION THREAD

**Discussion by ozgurerdogan** [acmesh-official/acme.sh#4690] [General]
I am using hestiacp and non default port (80-443)
I issued a cert with success with :
acme.sh --issue --dns dns_cf -d domain.com -w /home/monitor/web/domain.com/public_html

But then I needed to manually copy cert to site manually. So
1. Is it possible for acme.sh to auto bing cert to site's conf file?
2. Will I have to manually copy when it is renewed?

**Comment by ozgurerdogan** [comment]
Ok I found this title: https://github.com/acmesh-official/acme.sh#3-install-the-cert-to-apachenginx-etc

But must I manually run it after cert is issued? How about auto renewals?

**Comment by neilpang** [comment]
![image](https://github.com/acmesh-official/acme.sh/assets/8305679/635d737c-b7e0-4d13-99ad-1da26788a84c)

**Comment by ozgurerdogan** [comment]
"Install/copy the cert/key to the production Apache or Nginx path." this needs to be done manually?
